starkjs/run_norm_00.sh

```diff
@@ -0,0 +1,28 @@
+mkdir -p circuits && node fibonacci/fibonacci.js -w circuits -i "./input1.json" 
```

Comment:
it's better to move run_norm_00/1.sh to test directory?  cause it just for tests?